mapbox_gl_platform_interface/lib/src/mapbox_gl_platform_interface.dart

```diff
@@ -278,4 +278,7 @@ abstract class MapboxGlPlatform {
     throw UnimplementedError(
         'getMetersPerPixelAtLatitude() has not been implemented.');
   }
+
+  void dispose() {
```

Comment:
might want to convert this to be abstract

Reply:
The class is abstract already